code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/woodcutter.dm

```diff
@@ -1,7 +1,7 @@
 /datum/advclass/woodcutter
-	name = "Woodcutter"
-	tutorial = "You are a strong Lumberjack, armed with an axe, you can gather wood \
-	either for yourself, or for others. You are a novice carpenter too, so you can bend wood into some items you need, and chop it efficiently."
+	name = "Woodworker"
```

Comment:
I'd also rather you either keep the name as it was or change all "woodcutter" references in code to "woodworker" (but i'd still personally would rather keep the name)

Reply:
i personally liked "Woodsman"